packages/babel-parser/benchmark/many-empty-statements/attachComment-false.bench.mjs

```diff
@@ -0,0 +1,19 @@
+import Benchmark from "benchmark";
+import baseline from "@babel-baseline/parser";
+import current from "../../lib/index.js";
+import { report } from "../util.mjs";
+
+const suite = new Benchmark.Suite();
+
+function benchCases(name, implementation, options) {
+  for (const length of [256, 512, 1024, 2048]) {
+    suite.add(`${name} ${length} empty statement`, () => {
+      implementation.parse(";".repeat(length), options);
```

Comment:
The performance of `many-empty-statements` is largely depended on `Node` creation/finalization and the complexity of tokenizer `next()`. I make it a folder as we can add more benchmark cases here.